criteria.md

```diff
@@ -0,0 +1,35 @@
+| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. | 1 (sandbox)    |
+| Build & Release | All software releases MUST be assigned a standardized unique version identifier. | 1 (sandbox)    |
+| Build & Release | Any websites related to the project development and release MUST be delivered using SSH or HTTPS and TLS. | 1 (sandbox)    |
+| Build & Release | All software releases MUST include a descriptive log of functional and security modifications. | 2 (incubating) |
+| Build & Release | All software releases MUST incorporate a policy to address applicable Software Composition Analysis results. | 3 (graduated)  |
+| Documentation   | The project documentation MUST include a policy for coordinated vulnerability reporting, with a clear timeframe for response. | 2 (incubating) |
+| Documentation   | The project documentation MUST include a mechanism for reporting defects. | 2 (incubating) |
+| Documentation   | The project MUST have one or more mechanisms for public discussions about proposed changes and usage obstacles. | 1 (sandbox)    |
+| Documentation   | The project documentation MUST include a guide for contributors and approvers that includes the requirements for acceptable contributions. | 2 (incubating) |
+| Documentation   | The project documentation MUST define a cadence in which known vulnerabilities are evaluated, and exploitable vulnerabilities are either fixed or verified as unexploitable. | 3 (graduated)  |
+| Documentation   | The project documentation MUST provide user guides for all basic functionality. | 1 (sandbox)    |
+| Documentation   | The project documentation MUST provide design documentation demonstrating all actions and actors within the system. | 2 (incubating) |
+| Documentation   | The project documentation MUST provide reference documentation that describes the external application programmer and human interface (both input and output) of the software                | 3 (graduated)  |
+| Documentation   | The project documentation MUST include an explanation of the contribution process. | 1 (sandbox)    |
+| Quality         | A version control system MUST be publicly readable and has a static URL. | 1 (sandbox)    |
+| Quality         | The version control system MUST contain a publicly readable record of all changes made, who made the changes, and when the changes were made. | 1 (sandbox)    |
+| Quality         | All released software assets MUST be associated with a machine-readable list of all direct and transitive dependencies with their associated version identifier. | 2 (incubating) |
+| Quality         | Any automated commit status checks MUST pass or require manual intervention prior to merge. | 2 (incubating) |
+| Quality         | All subprojects MUST enforce security requirements as applicable to the status and intent of the subproject. | 2 (incubating) |
```

Comment:
I'm having trouble parsing this. Is there another way to say it?

Reply:
```suggestion
| Quality         | All codebases within the project MUST enforce security requirements as applicable to the status and intent of the codebase. | 2 (incubating) |
```

How's this?

cc @david-a-wheeler @SecurityCRob

Reply:
The idea behind the term "subproject" here is that a project may contain multiple codebases, which would be covered by all the other criteria... but we should address the fact that subordinate elements themselves function as independent projects with delineated leadership and processes— in those cases, the subprojects themselves will need to meet all of the criteria on their own.

This isn't an essential criteria— we can remove it if it's too difficult to hammer out.